src/ray/protobuf/gcs.proto

```diff
@@ -102,15 +102,17 @@ message TaskTableData {
 message ActorTableData {
   // State of an actor.
   enum ActorState {
+    // Actor is unresolved.
+    UNRESOLVED = 0;
```

Comment:
Could we call this something like `PENDING_DEPENDENCY_RESOLUTION`, and the other one `PENDING_CREATION`? `UNRESOLVED` is a bit vague.

Reply:
> Could we call this something like `PENDING_DEPENDENCY_RESOLUTION`, and the other one `PENDING_CREATION`? `UNRESOLVED` is a bit vague.

Okey I will fix it.